app/components/progress_tab/hypertension/diagnosis_report_component.html.erb

```diff
@@ -0,0 +1,41 @@
+<div id="hypertension-report" class="d-none">
+  <div class="mb-8px pt-16px pb-16px bgc-white bs-card">
+    <a
+      class="d-block w-24px h-24px mb-24px pl-16px"
+      href="#"
+      onclick="goToPage('hypertension-report', 'home-page'); return false;"
+    >
+      <div class="d-inline-block">
+        <%= inline_file("chevron-left.svg") %>
+      </div>
+    </a>
+    <div class="pr-16px pl-16px">
+      <h1 class="m-0px mb-8px fw-bold fs-24px">
+        <%= t("progress_tab.hypertension_report_title") %> report
+      </h1>
+      <p class="m-0px p-0px ta-left fw-normal fs-16px c-grey-dark">
+        <%= t("progress_tab.last_updated_at",
+              date: display_date(@last_updated_at),
```

Comment:
[non-blocking] I ran into this locally: If `last_updated_at` is not present in the redis cache, this breaks the view since `display_date` can't handle nil values. We can put this in a story and fix later: ideally missing redis keys shouldn't break anything on the view and `display_date` should show an empty string.

Reply:
Done

Reply:
Can you drop a link to the story here for posterity?